### PR TITLE
Card

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my component!

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
in the entire card.css file, make the indentation proper. So that it is easy to read for you and others too. Rest looks good

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+.tag{
+	position: absolute;
+	border-radius:6px;
+	padding: 4px 8px;
+	top: 5%;
+	left: 5%;
+	background-color: var(--text-color);
+	color: var(--bg-code);
+}
```
Comment (Naveenchandar) [inline Components/Card/card.css:59]: Please Indent the code.

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+.btn-dismiss{
+  position: absolute;
+  top: 5%;
+	right: 5%;
```
Comment (Naveenchandar) [inline Components/Card/card.css:64]: Remove extra space

File: Components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+.btn-dismiss:hover{
+  border: 2px solid var(--black-bg);
+}
+.card-horizontal {
+	display: grid;
+	grid-template-columns: 30% 70%;
+	position: relative;
+	width:36rem;
+  padding: 1.2rem;
+}
```
Comment (Naveenchandar) [inline Components/Card/card.css:80]: Indentation missing.

**Pull request merged**